sample/01-cats-app/src/cats/cats.controller.spec.ts

```diff
@@ -31,4 +31,21 @@ describe('CatsController', () => {
       expect(await catsController.findAll()).toBe(result);
     });
   });
+
+  describe('create', () => {
+    it('should add a new cat', async () => {
+      const cat: Cat = {
+        age: 2,
+        breed: 'Bombay',
+        name: 'Pixel',
+      };
+      const expectedCatArray = [cat];
+
+      expect(await catsController.findAll()).toStrictEqual([]);
```

Comment:
```suggestion
      await expect(catsController.findAll()).resolves.toStrictEqual([]);
```

can we use this instead? https://jestjs.io/docs/tutorial-async#asyncawait